Source/Svg.csproj

```diff
@@ -152,7 +152,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Fizzler" Version="1.1.0" />
+    <PackageReference Include="Fizzler" Version="1.1.0" Condition="'$(TargetFramework)'=='net35' Or '$(TargetFramework)'=='net452'"/>

```

Comment:
Is this change necessary ?
ref. https://github.com/vvvv/SVG/issues/563#issuecomment-524795988

Reply:
Not necessary, but Fizzler 1.1.0 is incompatible with NET Core 2

Reply:
So, it should not be included in this PR.